<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigPreviewWidget</class>
 <widget class="QWidget" name="ConfigPreviewWidget">
  <property name="windowTitle">
   <string>Preview</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="buildFormLayout">
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="buildLabel">
       <property name="whatsThis">
        <string>&lt;p&gt;Specify whether the preview should be generated automatically or manually.  If the option &quot;Automatically&quot; is checked, the TikZ code in the editor is compiled while editing.  If the option &quot;Manually&quot; is checked, a &quot;Build&quot; button is available and the TikZ code is only compiled when you press that button.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>Preview generation:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="buildAutomaticallyRadio">
       <property name="whatsThis">
        <string>&lt;p&gt;If this option is checked, the TikZ code in the editor is compiled while editing.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>&amp;Automatically</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buildButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="buildManuallyRadio">
       <property name="whatsThis">
        <string>&lt;p&gt;If this option is checked, a &quot;Build&quot; button is available and the TikZ code is only compiled when you press that button.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>&amp;Manually</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buildButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="coordinatesFormLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="showCoordinatesCheck">
       <property name="whatsThis">
        <string>&lt;p&gt;If this option is checked, the coordinates of the mouse cursor in the preview are shown.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>S&amp;how mouse coordinates:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="bestPrecisionRadio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="whatsThis">
        <string>&lt;p&gt;If this option is checked, the mouse coordinates are shown with enough precision but without redundant decimals.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>Use &amp;best precision</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">precisionButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="specifyPrecisionWidget" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="specifyPrecisionRadio">
          <property name="whatsThis">
           <string>&lt;p&gt;If this option is checked, the mouse coordinates are shown with the specified number of decimals.&lt;/p&gt;</string>
          </property>
          <property name="text">
           <string>Speci&amp;fy precision:</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">precisionButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="specifyPrecisionSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="whatsThis">
           <string>&lt;p&gt;Specify the number of decimals with which the mouse coordinates are shown.&lt;/p&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="whiteSpaceGridLayout">
     <item row="0" column="1">
      <widget class="ColorButton" name="backgroundColorButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="whatsThis">
        <string>&lt;p&gt;Select the color for the background.&lt;/p&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="colorBackgroundLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="whatsThis">
        <string>&lt;p&gt;Select the color for the background.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>Background Color</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QToolButton</extends>
   <header>../common/utils/colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>showCoordinatesCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>bestPrecisionRadio</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showCoordinatesCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>specifyPrecisionWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specifyPrecisionRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>specifyPrecisionSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buildButtonGroup"/>
  <buttongroup name="precisionButtonGroup"/>
 </buttongroups>
</ui>
